### PR TITLE
fix(hotfix): bottom sheet appearing when turning biometrics off and remove toast.

### LINKED ISSUE #969: Issues with biometric authentication (or rather when it is disabled)
### Steps to reproduce

When I turn *off* biometric authentication I still get this which is meaningless if it is being turned *off*:
Screenshot_20240307_220920_CloudStream Beta

I see this everytime I open the app, even if it is not enabled at all in CloudStream:
Screenshot_20240307_220827_CloudStream Beta

Also this should not say "fingerprint" as there are other types of biometric authentication.

It also shows the "Logged in with" toast before you actually complete login with biometric if enabled, it should wait to show that for the login to actually be completed.

### Expected behavior

...

### Actual behavior

...

### Cloudstream version and commit hash

4.3.1 bd69054

### Android version

Android 14

### Logcat

_No response_

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to pre-release version **[Latest](https://github.com/recloudstream/cloudstream/releases)**.
- [X] If related to a provider, I have checked the site and it works, but not the app.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by IndusAryan**
this needs some talk in the issue before being undrafted

closes #969

### CODE REVIEW

File: app/src/main/java/com/lagradost/cloudstream3/MainActivity.kt [hunk 1]
```diff
@@ -1222,8 +1221,6 @@ class MainActivity : AppCompatActivity(), ColorPickerDialogListener, BiometricAu
 
                 // hide background while authenticating, Sorry moms & dads 🙏
                 binding?.navHostFragment?.isInvisible = true
-            } else {
-                showToast(R.string.phone_not_secured, LENGTH_LONG)
```
Comment (Luna712) [inline app/src/main/java/com/lagradost/cloudstream3/MainActivity.kt:1226]: phone_not_secured message is now unused and can be removed from values also.

**Review by Luna712 — CHANGES_REQUESTED** [review]

**Review by Luna712 — APPROVED** [review]

**Comment by Luna712** [comment]
@IndusAryan I wonder if maybe you should have to confirm biometrics before it can even be enabled also? iE enter biometrics once when enabling and if it fails don't actually enable also.

**Comment by IndusAryan** [comment]
this was thought as well as tried, to get this it requires bringing interface onAuthenticationSuccess in the accountSetting, but it requiresActivity() to work and context, but now i have learned context more , will try again.

**Comment by Luna712** [comment]
> this was thought as well as tried, to get this it requires bringing interface onAuthenticationSuccess in the accountSetting, but it requiresActivity() to work and context, but now i have learned context more , will try again.

@IndusAryan 

This is probably something I could help with if you need/want?

**Comment by IndusAryan** [comment]
i will draft this and push changes you can review them, or co commit them.

**Comment by Luna712** [comment]
@IndusAryan can we unmark this as draft and do the rest in a separate PR, this is quite important to get merged for me, especially that random toast is pretty annoying as it even appears on TV which doesn't even support biometrics?

**Pull request merged**